ledger/src/bigtable_upload.rs

```diff
@@ -18,6 +18,9 @@ use {
 // Attempt to upload this many blocks in parallel
 const NUM_BLOCKS_TO_UPLOAD_IN_PARALLEL: usize = 32;
 
+// Number of blockstore threads to read from in parallel
+const NUM_BLOCKSTORE_THREADS: usize = 16;
```

Comment:
This can be a default, but we probably need to make this tunable.

Reply:
want me to pass in as Option<usize> to the fn with this as default if None?

Reply:
Or pass in a config struct to the fn that includes blockstore_threads and blocks_to_upload_in_parallel?
I'm surprised we haven't wanted to tune `NUM_BLOCKS_TO_UPLOAD_IN_PARALLEL` more easily before now, actually

Reply:
we typically default thread counts to a function of `num_cpus::get`

Reply:
added a config object, lmk what you think